go.mod

```diff
@@ -131,6 +131,7 @@ require (
 	github.com/cenkalti/backoff/v4 v4.3.0 // indirect
 	github.com/cespare/xxhash/v2 v2.3.0 // indirect
 	github.com/cloudflare/circl v1.6.0 // indirect
+	github.com/cmatthias/mapstructure v1.6.0 // indirect
```

Comment:
It is *highly* suspicious that anyone would depend on this! It's a private fork of an archived repo - which was popular, but isn't anymore.

Reply:
can you point me at an alternative?

Reply:
switched to github.com/go-viper/mapstructure/v2